tests/practice1_spec.rb

```diff
@@ -0,0 +1,24 @@
+require_relative '../practice1.rb'
+
+RSpec.describe "#DecReprSeniorCoding" do
+  it "returns the largest number in the family of N" do
```

Comment:
Normally we use single quote instead of double quote, please see https://github.com/rubocop/ruby-style-guide for the practical style guide kub